api/PaymentMethodChangeEvent.json

```diff
@@ -50,6 +50,58 @@
           "deprecated": false
         }
       },
+      "PaymentMethodChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentMethodChangeEvent/PaymentMethodChangeEvent",
+          "spec_url": "https://w3c.github.io/payment-request/#dom-paymentmethodchangeevent-constructor",
+          "description": "<code>PaymentMethodChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": "76"
+            },
+            "chrome_android": {
+              "version_added": "76"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Available only in nightly builds. Requires <code>dom.payments.request.enabled</code> to be set to <code>true</code> and the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "Available only in nightly builds. Requires <code>dom.payments.request.enabled</code> to be set to <code>true</code> and the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "63"
+            },
+            "opera_android": {
+              "version_added": "54"
+            },
+            "safari": {
+              "version_added": "12.1"
```

Comment:
The description gave evidence for Chromium, is this also based on source inspection, or testing? https://mdn-bcd-collector.appspot.com/tests/api/PaymentMethodChangeEvent confirms it's currently supported at least.

(Note that mdn-bcd-collector could probably fill in most of this data, avoiding the need to look at source code.)

Reply:
Never mind, I've confirmed this with https://github.com/foolip/mdn-bcd-collector#updating-bcd-using-the-results and let's assume iOS matches.

Reply:
So, kept as it i.